### PR TITLE
IW-1498 | Exposing data for search suggestions tracking

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/iwing

### CODE REVIEW

File: app/components/global-navigation/search.js [hunk 1]
```diff
@@ -34,6 +35,7 @@ export default Component.extend({
 	},
 	/* eslint-disable ember/avoid-leaking-state-in-ember-objects */
 	isEmptyQuery: empty('state.query'),
+	searchId: null,
```
Comment (rybmat) [inline app/components/global-navigation/search.js]: in suggestions events there is no such thing like `searchId`, you should name it `suggestionId `

File: app/components/global-navigation.js [hunk 2]
```diff
@@ -173,6 +173,11 @@ export default Component.extend({
 			this.onSearchSuggestionChosen(suggestion);
 		},
 
+		onSearchSuggestionsImpression(suggestions, searchId) {
+			// eslint-disable-next-line no-console
+			console.info('Suggestions impression', suggestions, 'with searchId', searchId);
```
Comment (rybmat) [inline app/components/global-navigation.js]: Why we need console.log here?

Comment (bkoval) [reply app/components/global-navigation.js]: This is definitely not the place for console.log, it was supposed to land in a controller of the dummy app :)

File: app/components/global-navigation/search.js [hunk 3]
```diff
@@ -92,7 +94,9 @@ export default Component.extend({
 					return response.json().then((data) => {
 						const suggestions = data.suggestions.map((suggestion) => {
 							return EmberObject.create({
-								title: suggestion
+								title: suggestion,
+								articleId: data.ids && data.ids[suggestion],
+								query: data.query,
```
Comment (rybmat) [inline app/components/global-navigation/search.js]: I think this field should be named `value`

**Pull request merged**